sound/soc/sof/pcm.c

```diff
@@ -484,6 +489,9 @@ static int sof_pcm_close(struct snd_soc_component *component,
 		 */
 	}
 
+	if (pcm_ops->close)
```

Comment:
Ack here as well, this belongs to the generic commit.

Reply:
sure, thanks